types/gatherer.d.ts

```diff
@@ -45,7 +45,7 @@ declare global {
       /** The cached results of computed artifacts. */
       computedCache: Map<string, ArbitraryEqualityMap>;
       /** The set of available dependencies requested by the current gatherer. */
-      dependencies: TDependencies extends DefaultDependenciesKey ?
+      dependencies: [TDependencies] extends [DefaultDependenciesKey] ?
```

Comment:
`DefaultDependenciesKey` isn't actually a union, just an alias, so I wouldn't [expect to need brackets around that](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgGIHt3IN4ChkHIAeAXMiAK4C2ARtPoQJ5mW32HIBeL1dUuAX1y4wjAA4oACugDOM4DQA2EACrIAvMgDWERuhhpMyAD7IA5AH0Q6EBAtnhoicgBCcKAB41EIpBAATGWRpOQVlFQA+DWDgBC0PDHQAGmQAUSIERQp-CC8Uy2tbewiIx3EUN04vZB8-QODZeSVVKM0AbRUAXRrfCACggps7M2QAfhwBZDJJWPjElPTM7NyVfKsh4tLhBBsZMGQ4MjdPMyIR0zNGM1acUmQARhTmB6EdkD3kGiO4KtPz8yuN2wd0eyGe9yEuDgADoiNF7lDoYx4cIaLCUWjkZp7kA).
```suggestion
      dependencies: [TDependencies] extends DefaultDependenciesKey ?
```